docs/src/frontend/index.md

```diff
@@ -384,11 +384,20 @@ stdlib.fundFromFaucet(account, balance) => Promise<void>
 ```
 
 Adds the given balance of network tokens to a Reach account abstraction.
-This can only be used in private testing scenarios,
-as it uses a private faucet to issue network tokens,
-as well as certain public TestNet scenarios.
+This can only be used in private testing scenarios, as it uses a private faucet to issue network tokens, as well as certain public TestNet scenarios.
 You can use `{!js} canFundFromFaucet` to check if `{!js} fundFromFaucet` can be used.
 
+Example:
+
+```js
+load: /examples/js-tests/index.mjs
+md5: 389bd098127db61d413f7586e81b561f
+range: 324 - 325
+```
+
+This code creates an account, and then funds that account with 100000 units from a private faucet.
```

Comment:
We now have this beautiful functionality where we can add "_" underscores to numbers that are longer than three digits. :)
I suggest you update this source code with that feature.
In the docs, use a comma. `100,000`